app.py

```diff
@@ -1,23 +1,19 @@
-from flask import Flask, render_template, request, flash, redirect, url_for, make_response
+from flask import Flask, render_template, request, flash, redirect, url_for, make_response, g, session
```

Comment:
Are you using `session`?

Reply:
Kind of! I used it in 'comment' feature which we removed in newer version :))

Reply:
ok I don't see why you'd use that for activity logs, but let's remove it for now since we're not using it. Also remove the remaining traces of the comment feature.

Reply:
Sure